crates/db_views_actor/src/comment_reply_view.rs

```diff
@@ -217,6 +217,11 @@ fn queries<'a>() -> Queries<
       CommentSortType::Top => query.order_by(comment_aggregates::score.desc()),
     };
 
+    // Don't show replies from blocked persons
+    if let Some(my_person_id) = options.my_person_id {
+      query = query.filter(not(is_creator_blocked(my_person_id)));
+    }
+
```

Comment:
Does this file need a left join with person_block like the other file does?

Reply:
is_creator_blocked is defined as a subselect on line 81

Reply:
Ya, the other one had it missing for some reason.